src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -186,6 +186,12 @@ void handleDirectCommand(Channel channel, User sender, String payload) {
             return;
         }
 
+        m = Pattern.compile("(?:remove|revoke) (\\S+)(?: as)? (a )?(committ?er|member).*",CASE_INSENSITIVE).matcher(payload);
```

Comment:
I think you need something like the `on (.*)` here to get the `justForThisRepo` value.

Reply:
Right, reads weird otherwise, good catch 👍🏻